core-web/apps/dotcms-ui/src/app/view/directives/dot-avatar/dot-avatar.directive.spec.ts

```diff
@@ -40,12 +41,14 @@ describe('DotAvatarDirective', () => {
         component.text = 'test';
         fixture.detectChanges();
 
-        setTimeout(() => {
-            fixture.detectChanges();
-            expect(element.query(By.css('p-avatar')).nativeElement.textContent).toBe(
-                component.text.charAt(0).toUpperCase()
-            );
-            done();
-        }, 100);
+        fixture.whenStable().then(() => {
+            setTimeout(() => {
```

Comment:
Is this `setTimeout` still needed?

Reply:
I wrap the code inside the `whenStable` because was falling.